templates/etc/init.d/deb/td-agent

```diff
@@ -202,7 +203,7 @@ do_configtest() {
 RETVAL=0
 case "$1" in
 "start" )
-  echo -n "Starting ${TD_AGENT_NAME}: "
+  echo -n "Starting ${TD_AGENT_VERSION}: "
```

Comment:
I was being lazy in my variable naming. `TD_AGENT_VERSION` actually includes both name and version.